### ISSUE TITLE
namelist.fire setting

### ISSUE THREAD

**Issue by zli867** [openwfm/WRF-SFIRE#82]
Dear all,

I am confused about the namelist.fire which is one of the input for the model. It seems fgi assigns mass loading of surface fuel for each fuel categoryies But the default fgi is looks like:
https://github.com/openwfm/WRF-SFIRE/blob/f2dd6292cf96222d7851433f4243fb97980e4779/test/em_fire/namelist.fire#L29

It does not include all fuel categories (only 1-7). I used this namelist to run a case, the log file for the WRF-SFIRE case shows some burned mass for the Hardwood litter (9), Timber (litter + understor) (10). I am confused about why it happens.

Also, if I want all fuel categories to have their own fgi, should I uncomment the line:
https://github.com/openwfm/WRF-SFIRE/blob/f2dd6292cf96222d7851433f4243fb97980e4779/test/em_fire/namelist.fire#L30

I can put part of the log for review.
```
SFIRE: domain  1
SFIRE:Time   86399.000 s Average surface wind   0.964E-02 m/s
SFIRE:Time   86399.000 s Maximum surface wind   0.438E+01 m/s
SFIRE:Time   86399.000 s Fire area              0.306E+07 m^2
SFIRE:Time   86399.000 s Heat output            0.107E+09 W
SFIRE:Time   86399.000 s Max heat flux          0.367E+04 W/m^2
SFIRE:Time   86399.000 s Latent heat output     0.109E+08 W
SFIRE:Time   86399.000 s Max latent heat flux   0.375E+03 W/m^2
SFIRE:add_fire_emissions: chem_opt=  0 tracer_opt=  2 num_chem=  0 num_tracer=  9 max tracer=  9
SFIRE:array tracer has dimensions    1: 57   1: 35   1: 46   1:  9
SFIRE:array chem has dimensions    1: 57   1: 35   1: 46   1:  0
SFIRE: tracer   tr17_1   tr17_2   tr17_3   tr17_4   tr17_5
SFIRE:Pointer        2        3        4        5        6
SFIRE: tracer   tr17_6   tr17_7   tr17_8
SFIRE:Pointer        7        8        9
SFIRE:Fire tracers into atmosphere level  1
SFIRE:Inserting chemical species
SFIRE:Computing totals over all processes
SFIRE:Computing totals of tracers over all processes
SFIRE:Total emissions in g
SFIRE:Emissions      tr17_1     tr17_2     tr17_3     tr17_4     tr17_5
SFIRE:Cumulative  0.292E+11  0.367E+10  0.115E+09  0.814E+09  0.128E+10
SFIRE:Rate per s  0.690E+05  0.866E+04  0.280E+03  0.192E+04  0.303E+04
SFIRE:
SFIRE:Emissions      tr17_6     tr17_7     tr17_8
SFIRE:Cumulative  0.117E+09  0.140E+10  0.469E+06
SFIRE:Rate per s  0.277E+03  0.328E+04  0.112E+01
SFIRE:
SFIRE:1: Short grass (1 ft)          burned    0.3963E+06kg, rate    0.2695     kg/s
SFIRE:2: Timber (grass and understor burned    0.2789E+07kg, rate     2.371     kg/s
SFIRE:3: Tall grass (2.5 ft)         burned    0.2190E+08kg, rate     61.04     kg/s
SFIRE:5: Brush (2 ft)                burned    0.1749E+07kg, rate    0.4585E-01 kg/s
SFIRE:7: Southern rough              burned    0.1509E+07kg, rate     13.49     kg/s
SFIRE:8: Closed timber litter        burned    0.2161E+09kg, rate     417.8     kg/s
SFIRE:9: Hardwood litter             burned    0.5293E+08kg, rate     212.4     kg/s
SFIRE:10: Timber (litter + understor burned    0.8621E+06kg, rate    0.1977E-03 kg/s
SFIRE:                    Total fuel burned    0.2983E+09kg, rate     707.4     kg/s
Timing for sfire:    0.16124 elapsed seconds
++++++CALL ERROB AT KTAU = 86399 AND INEST =  1:  NSTA =     0 ++++++
 OBS NUDGING FOR IN,J,KTAU,XTIME,IVAR,IPL:  1 10 86399  1439.98  1  1 rindx=****
 OBS NUDGING FOR IN,J,KTAU,XTIME,IVAR,IPL:  1 10 86399  1439.98  2  2 rindx=****
Timing for main: time 2021-03-21_00:00:00 on domain   1:    0.38112 elapsed seconds
Timing for Writing wrfout_d01_2021-03-21_00:00:00 for domain        1:    2.95722 elapsed seconds
d01 2021-03-21_00:00:00 wrf: SUCCESS COMPLETE WRF
```
Thanks,
Zongrun

**Comment by zli867** [comment]
Sorry for bothering you, the comments do not affect the second line. I will close the issue.

**Comment by adamk0** [comment]
Zongrun,
The fuel load fgi is provided in two lines, the middle one is commented out.
fgi =  0.166, 0.897, 0.675, 2.468, 0.785, 1.345, 1.092,
!jc fgi=  0.166, 0.897, 1.076, 2.468, 0.785, 1.345, 1.092
1.121, 0.780, 2.694, 2.582, 7.749, 13.024, 1.e-7,

So effectively it is the same as:
fgi =  0.166, 0.897, 0.675, 2.468, 0.785, 1.345, 1.092, 1.121, 0.780, 2.694, 2.582, 7.749, 13.024, 1.e-7

One fuel load value per each fuel category.

Adam Kochanski

Associate Professor
SJSU Wildfire Interdisciplinary Research Center




From: Zongrun ***@***.***>
Date: Thursday, October 5, 2023 at 11:41 AM
To: openwfm/WRF-SFIRE ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [openwfm/WRF-SFIRE] namelist.fire setting (Issue #82)

Dear all,

I am confused about the namelist.fire which is one of the input for the model. It seems fgi assigns mass loading of surface fuel for each fuel categoryies But the default fgi is looks like:
https://github.com/openwfm/WRF-SFIRE/blob/f2dd6292cf96222d7851433f4243fb97980e4779/test/em_fire/namelist.fire#L29<https://www.google.com/url?q=https://github.com/openwfm/WRF-SFIRE/blob/f2dd6292cf96222d7851433f4243fb97980e4779/test/em_fire/namelist.fire%23L29&source=gmail-imap&ust=1697136112000000&usg=AOvVaw0_WuB0dkfIGSojDYsXvJ1Y>

It does not include all fuel categories (only 1-7). I used this namelist to run a case, the log file for the WRF-SFIRE case shows some burned mass for the Hardwood litter (9), Timber (litter + understor) (10). I am confused about why it happens.

Also, if I want all fuel categories to have their own fgi, should I uncomment the line:
https://github.com/openwfm/WRF-SFIRE/blob/f2dd6292cf96222d7851433f4243fb97980e4779/test/em_fire/namelist.fire#L30<https://www.google.com/url?q=https://github.com/openwfm/WRF-SFIRE/blob/f2dd6292cf96222d7851433f4243fb97980e4779/test/em_fire/namelist.fire%23L30&source=gmail-imap&ust=1697136112000000&usg=AOvVaw3E4bl5q2MKUl3HlfS82rXx>

I can put part of the log for review.

SFIRE: domain  1

SFIRE:Time   86399.000 s Average surface wind   0.964E-02 m/s

SFIRE:Time   86399.000 s Maximum surface wind   0.438E+01 m/s

SFIRE:Time   86399.000 s Fire area              0.306E+07 m^2

SFIRE:Time   86399.000 s Heat output            0.107E+09 W

SFIRE:Time   86399.000 s Max heat flux          0.367E+04 W/m^2

SFIRE:Time   86399.000 s Latent heat output     0.109E+08 W

SFIRE:Time   86399.000 s Max latent heat flux   0.375E+03 W/m^2

SFIRE:add_fire_emissions: chem_opt=  0 tracer_opt=  2 num_chem=  0 num_tracer=  9 max tracer=  9

SFIRE:array tracer has dimensions    1: 57   1: 35   1: 46   1:  9

SFIRE:array chem has dimensions    1: 57   1: 35   1: 46   1:  0

SFIRE: tracer   tr17_1   tr17_2   tr17_3   tr17_4   tr17_5

SFIRE:Pointer        2        3        4        5        6

SFIRE: tracer   tr17_6   tr17_7   tr17_8

SFIRE:Pointer        7        8        9

SFIRE:Fire tracers into atmosphere level  1

SFIRE:Inserting chemical species

SFIRE:Computing totals over all processes

SFIRE:Computing totals of tracers over all processes

SFIRE:Total emissions in g

SFIRE:Emissions      tr17_1     tr17_2     tr17_3     tr17_4     tr17_5

SFIRE:Cumulative  0.292E+11  0.367E+10  0.115E+09  0.814E+09  0.128E+10

SFIRE:Rate per s  0.690E+05  0.866E+04  0.280E+03  0.192E+04  0.303E+04

SFIRE:

SFIRE:Emissions      tr17_6     tr17_7     tr17_8

SFIRE:Cumulative  0.117E+09  0.140E+10  0.469E+06

SFIRE:Rate per s  0.277E+03  0.328E+04  0.112E+01

SFIRE:

SFIRE:1: Short grass (1 ft)          burned    0.3963E+06kg, rate    0.2695     kg/s

SFIRE:2: Timber (grass and understor burned    0.2789E+07kg, rate     2.371     kg/s

SFIRE:3: Tall grass (2.5 ft)         burned    0.2190E+08kg, rate     61.04     kg/s

SFIRE:5: Brush (2 ft)                burned    0.1749E+07kg, rate    0.4585E-01 kg/s

SFIRE:7: Southern rough              burned    0.1509E+07kg, rate     13.49     kg/s

SFIRE:8: Closed timber litter        burned    0.2161E+09kg, rate     417.8     kg/s

SFIRE:9: Hardwood litter             burned    0.5293E+08kg, rate     212.4     kg/s

SFIRE:10: Timber (litter + understor burned    0.8621E+06kg, rate    0.1977E-03 kg/s

SFIRE:                    Total fuel burned    0.2983E+09kg, rate     707.4     kg/s

Timing for sfire:    0.16124 elapsed seconds

++++++CALL ERROB AT KTAU = 86399 AND INEST =  1:  NSTA =     0 ++++++

 OBS NUDGING FOR IN,J,KTAU,XTIME,IVAR,IPL:  1 10 86399  1439.98  1  1 rindx=****

 OBS NUDGING FOR IN,J,KTAU,XTIME,IVAR,IPL:  1 10 86399  1439.98  2  2 rindx=****

Timing for main: time 2021-03-21_00:00:00 on domain   1:    0.38112 elapsed seconds

Timing for Writing wrfout_d01_2021-03-21_00:00:00 for domain        1:    2.95722 elapsed seconds

d01 2021-03-21_00:00:00 wrf: SUCCESS COMPLETE WRF

Thanks,
Zongrun
